client/src/js/users/components/__tests__/Group.test.js

```diff
@@ -1,5 +1,5 @@
 import { ListGroupItem } from "../../../base";
-import { UserGroup } from "../Group";
+import { Groups, UserGroup } from "../Group";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Groups' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4661579)